### PR TITLE
Refine expiration/retention policy for import tasks

### PR DESCRIPTION
**Pull request by soothing-rain**
Signed-off-by: Yuchen Gao <yuchen.gao@zilliz.com>

### CODE REVIEW

File: internal/rootcoord/import_manager.go [hunk 1]
```diff
@@ -574,59 +596,46 @@ func (m *importManager) loadFromTaskStore() error {
...
-// storeImportTask signs a lease and saves import task info into Etcd with this lease.
-func (m *importManager) storeImportTask(task *datapb.ImportTaskInfo) error {
-	log.Debug("saving import task to Etcd", zap.Int64("task ID", task.GetId()))
-	// Sign a lease. Tasks will be stored for at least `ImportTaskRetention` seconds.
-	leaseID, err := m.taskStore.Grant(int64(Params.RootCoordCfg.ImportTaskRetention))
-	if err != nil {
-		log.Error("failed to grant lease from Etcd for data import",
-			zap.Int64("task ID", task.GetId()),
+// updateImportTask stores or updates the import task info in Etcd.
+func (m *importManager) updateImportTask(ti *datapb.ImportTaskInfo) error {
```
Comment (yhmo) [inline internal/rootcoord/import_manager.go]: "updateImportTask", I think this method name is not proper, suggest change to "persistTaskInfo". In this class all the tasks are import task, no need to always say "ImportTask"

File: internal/rootcoord/import_manager.go [hunk 2]
```diff
@@ -574,59 +596,46 @@ func (m *importManager) loadFromTaskStore() error {
...
-// updateImportTaskStore updates the task info in Etcd according to task ID. It won't change the lease on the key.
-func (m *importManager) updateImportTaskStore(ti *datapb.ImportTaskInfo) error {
-	log.Debug("updating import task info in Etcd", zap.Int64("Task ID", ti.GetId()))
-	if taskInfo, err := proto.Marshal(ti); err != nil {
-		log.Error("failed to marshall task info proto", zap.Int64("Task ID", ti.GetId()), zap.Error(err))
-		return err
-	} else if err = m.taskStore.SaveWithIgnoreLease(BuildImportTaskKey(ti.GetId()), string(taskInfo)); err != nil {
-		log.Error("failed to update import task info info in Etcd", zap.Int64("Task ID", ti.GetId()), zap.Error(err))
+// removeImportTask removes the task info from Etcd.
+func (m *importManager) removeImportTask(tID int64) error {
```
Comment (yhmo) [inline internal/rootcoord/import_manager.go]: suggest change to "unpersistTaskInfo"

File: internal/rootcoord/import_manager.go (hunk 1 above)
Comment (soothing-rain) [reply internal/rootcoord/import_manager.go]: done.

File: internal/rootcoord/import_manager.go (hunk 2 above)
Comment (soothing-rain) [reply internal/rootcoord/import_manager.go]: changed to `yieldTaskInfo`

**Pull request merged**